Packs/MicrosoftExchangeOnline/ReleaseNotes/1_5_9.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### EWS O365
+
+Fixed an issue where attachments with invalid characters in the From header were not being processed correctly.
```

Comment:
Please be more specific - the invalid characters will now be replaced by white space character as to allow processing these mails.